proxy/http/HttpTransact.cc

```diff
@@ -1963,7 +1963,7 @@ HttpTransact::OSDNSLookup(State *s)
 
   char addrbuf[INET6_ADDRSTRLEN];
   TxnDebug("http_trans", "DNS lookup for O.S. successful IP: %s",
-           ats_ip_ntop(&s->server_info.dst_addr.sa, addrbuf, sizeof(addrbuf)));
+           ats_ip_nptop(&s->server_info.dst_addr.sa, addrbuf, sizeof(addrbuf)));
```

Comment:
If you're making this change, you also need to change the buffer to `INET6_ADDRPORTLEN`.

Reply:
Good catch. Looks like `ip_port_text_buffer` is a type we've created for this. I'll use that and update the other places in here to use ats_ip_nptop as well...the extra port information can be helpful.